docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -23,7 +23,8 @@ mathematical properties`_.
     provide more security, currently ``1024`` and below are considered
     breakable, and ``2048`` or ``4096`` are reasonable default key sizes for
     new keys. The ``public_exponent`` indicates what one mathematical property
-    of the key generation will be, ``65537`` should almost always be used.
+    of the key generation will be. Unless you have a special requirement for
```

Comment:
How about: "Unless you have a specific reason to do otherwise, you should always use 65537"